generated/php/google-cloud-error-reporting-v1beta1/proto/src/Google/Devtools/Clouderrorreporting/V1beta1/DeleteEventsRequest.php

```diff
@@ -11,7 +11,7 @@
 /**
  * Deletes all events in the project.
  *
- * Protobuf type <code>Google\Devtools\Clouderrorreporting\V1beta1\DeleteEventsRequest</code>
+ * Generated from protobuf message <code>google.devtools.clouderrorreporting.v1beta1.DeleteEventsRequest</code>
```

Comment:
Why do you change the package path here? Isn't the path separated by "\" in PHP?